Beginning DBM recommendations docs. (#26091)
DataDog/documentation: content/en/database_monitoring/_index.md

{{< img src="database_monitoring/databases-list.png" alt="The Databases page in Datadog" style="width:90%;" >}}

### View recommendations for guidance on what optimization and why

> The [Recommendations page][6] brings problems and optimization opportunities to your attention, saving time by telling you about what's most important to look at.

Reviewer: ```suggestion
The [Recommendations page][6] highlights problems and optimization opportunities, helping you save time by prioritizing what's most important.
```

Author: Done.
